### ISSUE TITLE
expand scope?

### ISSUE THREAD

**Issue by AlJohri** [ofek/userpath#5]
@ofek just wanted to get your thoughts on potentially expanding the scope (whether in a separate package or the same) to not just PATH modifications but appending arbitrary lines to shell profile.

just thinking about these kinds of snippets I see all over the place:

https://github.com/thoughtbot/laptop/blob/581e50e6a978c5b2d2c68134541437b02d6108d5/mac#L14-L31

**Comment by ofek** [comment]
Yes I've thought of doing that in a separate package but my time is limited.

Fyi I'm fixing #3 atm 😄
